tools/benchmark/binsize/src/main.rs

```diff
@@ -0,0 +1,26 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::fs;
+
+fn wasm_filesize(dir: &str) -> Result<u64, std::io::Error> {
+    let paths = fs::read_dir(dir).expect("Directory wasm/ not found!");
+    for path in paths {
+        let p = path.unwrap().path();
+        if let Some(suffix) = p.extension() {
+            if suffix == "wasm" {
+                println!(
+                    "{{\"biggerIsBetter\":false,\"name\":{:?},\"unit\":\"bytes\",\"value\":{}}}",
+                    p.file_stem().unwrap(),
+                    p.metadata()?.len()
+                );
+            }
+        }
+    }
+    Ok(1)
+}
+
+fn main() {
+    let _rc = wasm_filesize("wasmpkg");
```

Comment:
Question: Does it make sense tracking the gzipped size of the wasm as well? If download size is a concern, the compression rate is important.

e.g.
```
[ 18K]  ./borrowed_pattern.wasm
[ 13K]  ./wasm-opt/borrowed_pattern+opt.wasm
[6.2K]  ./wasm-opt/borrowed_pattern+opt.wasm.gz
```

Reply:
Good question. I don't have an answer, the underlying ticket doesn't mention it.
Will put this up for  'Discussion' in the next meeting.

Reply:
Ticket#912: https://github.com/unicode-org/icu4x/issues/912